src/chainlit/frontend/src/components/organisms/FormInput.tsx

```diff
@@ -0,0 +1,60 @@
+import Switch, { SwitchProps } from 'components/atoms/switch';
```

Comment:
Shouldn't we group the inputs together in an inputs directory? They are currently in different directories or is there a reason for doing it this way?

Reply:
Yes it can be, it can be an issue for later